### PR TITLE
Pulse-152 System Integration test

### PR DESCRIPTION
**Pull request by shashi4569**
Runs all pulse services(Log-collector, Alert-Engine, Colllection-roller), Starts example application, asserts records in solr collection and cleans up all services.

### CODE REVIEW

File: system-test/system-test-script.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  # killing_services
+  kill_service
+  echo "Sevices are terminated/killed"
+}
+
+trap cleanup EXIT
+mkdir -p /tmp/pulse-system-test
+
+# Writing logs to local directory
+export collection_roller_log="system-test/log_files/collectionroller.log"
```
Comment (afoerster) [inline system-test/system-test-script.sh]: Separate names with underscored like `collection_roller.log`. Also on the next four lines.

File: system-test/system-test-script.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+query_response=$(curl -i -o - --silent -X GET -u ${SOLR_USR}:${SOLR_PWD} "http://master3.valhalla.phdata.io:8983/solr/logging-pulse-test_latest/select?q=*%3A*&wt=json&indent=true")
+http_status_collection=$(echo "$query_response" | grep HTTP |  awk '{print $2}')
+
+echo $http_status_collection
+
+# Checking if the collection exists and if documents are collected
+
+if [[ "$http_status_collection" == 200 ]]; then
+       if [[ "query_response" =~ "\"numFound\":0" ]]; then
+                echo "Records assertion in Solr collection Failed!"
```
Comment (afoerster) [inline system-test/system-test-script.sh]: Should `exit 1` here so processes that use this script know the test has failed

File: system-test/system-test-script.sh [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+       else
+                echo "Records assertion in Solr collection Passed!"
+       fi
+else
+        echo "Records assertion in Solr collection Failed"
```
Comment (afoerster) [inline system-test/system-test-script.sh]: `exit 1` here again.

File: system-test/system-test-script.sh [hunk 4]
```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```
Comment (afoerster) [inline system-test/system-test-script.sh]: calling this just `system-test` is good, no need for `script`

File: system-test/integration-test-alert-engine.yml [hunk 5]
```diff
@@ -0,0 +1,30 @@
+---
```
[the remark this answers is not in the document]
Comment (MariyamG) [reply system-test/integration-test-alert-engine.yml]: resolved

File: system-test/integration-test-collection-roller.yml [hunk 6]
```diff
@@ -0,0 +1,11 @@
+---
```
[the remark this answers is not in the document]
Comment (MariyamG) [reply system-test/integration-test-collection-roller.yml]: fixed

File: system-test/integration-test-collection-roller.yml [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+solrConfigSetDir: /etc/pulse-logging/solr-configs/
```
[the remark this answers is not in the document]
Comment (MariyamG) [reply system-test/integration-test-collection-roller.yml]: fixed

File: system-test/system-test-script.sh [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+# Stopping the script if any command fails
+set -e
+source bin/env.sh
+
+function cleanup {
+  echo "Removing /tmp/pulse-system-test"
+  rm  -r /tmp/pulse-system-test
+  # killing_services
```
[the remark this answers is not in the document]
Comment (MariyamG) [reply system-test/system-test-script.sh]: added a more descriptive comment

File: system-test/system-test-script.sh [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+  kill_service
```
[the remark this answers is not in the document]
Comment (MariyamG) [reply system-test/system-test-script.sh]: fixed

File: system-test/system-test-script.sh (hunk 1 above)
Comment (MariyamG) [reply system-test/system-test-script.sh]: fixed

File: system-test/system-test-script.sh (hunk 2 above)
Comment (MariyamG) [reply system-test/system-test-script.sh]: fixed

File: system-test/system-test-script.sh (hunk 3 above)
Comment (MariyamG) [reply system-test/system-test-script.sh]: fixed

File: system-test/system-test-script.sh (hunk 4 above)
Comment (MariyamG) [reply system-test/system-test-script.sh]: fixed

**Pull request closed without merging**